Autopilot/Src/imu.cpp

```diff
@@ -45,7 +45,7 @@
 #define MAG_IF_3_DATA_MODE 0x02
 #define MAG_IF_2_DATA_MODE 0x4C
 #define MAG_IF_1_DATA_MODE 0x42
-#define MAG_REFRESH_50_HZ 0x07  // TODO mag should refresh at 200 Hz as well, but I cant seem to get that to work.
+#define MAG_REFRESH_50_HZ 0x07  // TODO mag should refresh at 200 Hz as well, but I cant seem to G that to work.
```

Comment:
If you could revert this back to how it was and exclude it from the PR that would be great. Simply copying and pasting the file (raw) from WARG/devel will do lol.

Reply:
can you just fix this typo in your code and recommit? If you do this file won't be included in your PR. 

replace the `G` with `get`